inference-engine/src/vpu/graph_transformer/src/graph_transformer.cpp

```diff
@@ -81,39 +89,36 @@ void CompileEnv::init(ncDevicePlatform_t platform, const PluginConfiguration& co
 #endif
 
     if (platform == ncDevicePlatform_t::NC_MYRIAD_2) {
-        g_compileEnv->config.compileConfig().hwOptimization = false;
+        g_compileEnv->config.set(ie::MYRIAD_ENABLE_HW_ACCELERATION, ie::PluginConfigParams::NO);
```

Comment:
I think you can remove this check at all, since Myriad2 is not supported

Reply:
I think this should be done in a separate PR on the removal of the PLATFORM option.

Reply:
There is such a [PR](https://github.com/openvinotoolkit/openvino/pull/6022).